sample_app/script/ci_build

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+dir=`readlink -f $0`
+dir="`dirname $dir`/.."
+RAILS_ROOT=`readlink -f $dir`
+cd $RAILS_ROOT
+
+bundle install --path=$RAILS_ROOT/.bundle
+
+for stage in test development production; do
+  PGPASSWORD=$POSTGRES_PASSWORD psql -h "$POSTGRES_HOST" -U "$POSTGRES_USER" -c "create database octopus_sample_app_$stage"
+done
+
+# Not working...
+bundle exec cucumber
```

Comment:
I attempted to get the `sample_app` to run, but cucumber fails with:
```
Deprecated: Found tags option '~@wip'. Support for '~@tag' will be removed from the next release of Cucumber. Please use 'not @tag' instead.
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/values/time_zone.rb:270: warning: circular argument reference - now
The PGconn, PGresult, and PGError constants are deprecated, and will be
removed as of version 1.0.

You should use PG::Connection, PG::Result, and PG::Error instead, respectively.

Called from /usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:236:in `load_dependency'
cannot load such file -- test/unit (LoadError)
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:251:in `require'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:251:in `block in require'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:236:in `load_dependency'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:251:in `require'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/railties-3.2.13/lib/rails/test_help.rb:5:in `<top (required)>'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:251:in `require'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:251:in `block in require'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:236:in `load_dependency'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/activesupport-3.2.13/lib/active_support/dependencies.rb:251:in `require'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-rails-1.4.5/lib/cucumber/rails.rb:11:in `<top (required)>'
/usr/src/app/sample_app/features/support/env.rb:7:in `require'
/usr/src/app/sample_app/features/support/env.rb:7:in `<top (required)>'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/glue/registry_and_more.rb:107:in `load'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/glue/registry_and_more.rb:107:in `load_code_file'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/runtime/support_code.rb:144:in `load_file'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/runtime/support_code.rb:85:in `block in load_files!'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/runtime/support_code.rb:84:in `each'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/runtime/support_code.rb:84:in `load_files!'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/runtime.rb:272:in `load_step_definitions'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/runtime.rb:68:in `run!'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/lib/cucumber/cli/main.rb:34:in `execute!'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/gems/cucumber-3.1.2/bin/cucumber:9:in `<top (required)>'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/bin/cucumber:23:in `load'
/usr/src/app/sample_app/.bundle/ruby/2.5.0/bin/cucumber:23:in `<top (required)>'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/cli/exec.rb:74:in `load'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/cli/exec.rb:74:in `kernel_load'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/cli/exec.rb:28:in `run'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/cli.rb:463:in `exec'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/cli.rb:27:in `dispatch'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/cli.rb:18:in `start'
/usr/local/lib/ruby/gems/2.5.0/gems/bundler-1.17.3/exe/bundle:30:in `block in <top (required)>'
/usr/local/lib/ruby/site_ruby/2.5.0/bundler/friendly_errors.rb:124:in `with_friendly_errors'
/usr/local/lib/ruby/gems/2.5.0/gems/bundler-1.17.3/exe/bundle:22:in `<top (required)>'
/usr/local/bundle/bin/bundle:23:in `load'
/usr/local/bundle/bin/bundle:23:in `<main>'
root@601d44653d95:/usr/src/app/sample_app# 
```

I'm not well versed on cucumber, so I'm not sure what to do here...

Reply:
@calh - the build is probably broken on the sample_app for a while - I think you can ignore this cucumber process for now.

Reply:
Ahh, good to know, thanks!